Makefile

```diff
@@ -0,0 +1,9 @@
+local-install:
```

Comment:
I'm usually against helper Makefiles in Flathub repos but it's up to you.

Reply:
inherited from old non-flathub flatpak repo, personal anti-forget-mechanism, removed.